cfme/configure/access_control.py

```diff
@@ -168,149 +238,298 @@ def description(self):
+    title = Text('#explorer_title_text')
+
+    group_order_selector = UpDownSelect(
+        '#seq_fields',
+        './/a[@title="Move selected fields up"]/img',
+        './/a[@title="Move selected fields down"]/img')
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+    cancel_button = Button('Cancel')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == "Editing Sequence of User Groups"
+        )
+
+
+class GroupEditTagsView(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    tag_table = Table("//div[@id='assignments_div']//table")
+
+    select_tag = BootstrapSelect(id='tag_cat')
+    select_value = BootstrapSelect(id='tag_add')
+
+    save_button = Button('Save')
+    cancel_button = Button('Cancel')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == 'Editing My Company Tags for "EVM Groups"'
+        )
 
 
 class Group(Updateable, Pretty, Navigatable):
-    group_form = Form(
-        fields=[
-            ('ldap_groups_for_user', AngularSelect("ldap_groups_user")),
-            ('description_txt', Input('description')),
-            ('lookup_ldap_groups_chk', Input('lookup')),
-            ('role_select', AngularSelect("group_role")),
-            ('group_tenant', AngularSelect("group_tenant"), {"appeared_in": "5.5"}),
-            ('user_to_look_up', Input('user')),
-            ('username', Input('user_id')),
-            ('password', Input('password')),
-        ])
     pretty_attrs = ['description', 'role']
 
     def __init__(self, description=None, role=None, tenant="My Company", user_to_lookup=None,
-            ldap_credentials=None, appliance=None):
+                 ldap_credentials=None, tag=None, host_cluster=None, vm_template=None,
+                 appliance=None):
         Navigatable.__init__(self, appliance=appliance)
         self.description = description
         self.role = role
         self.tenant = tenant
         self.ldap_credentials = ldap_credentials
         self.user_to_lookup = user_to_lookup
+        self.tag = tag
+        self.host_cluster = host_cluster
+        self.vm_template = vm_template
 
     def create(self):
-        navigate_to(self, 'Add')
-        fill(self.group_form, {'description_txt': self.description,
-                               'role_select': self.role,
-                               'group_tenant': self.tenant},
-             action=form_buttons.add)
-        flash.assert_success_message('Group "{}" was saved'.format(self.description))
+        view = navigate_to(self, 'Add')
+        view.fill({
+            'description_txt': self.description,
+            'role_select': self.role,
+            'group_tenant': self.tenant
+        })
+        if self.tag is not None:
```

Comment:
If you can add tags as a part of the creation dialog, I see no reason why to avoid it in create. If the tagging is done separately in separate window, then it should be a separate method.